src/apis/attsrv.ts

```diff
@@ -88,7 +87,7 @@ const attendeeDtoFromRegistrationInfo = (registrationInfo: RegistrationInfo): At
  *
  * This endpoint is optimized in the backend for high traffic, so it is safe to call during initial reg.
  */
-export const registrationCountdownCheck = () => ajax<CountdownDto | ErrorDto>({
```

Comment:
What happens if the call returns an ErrorDto? In some cases you may wish to have access to the detailed error messages so you can display them to the user.

Reply:
It raises an error, which is untyped.